### PR TITLE
Feature: Added docs for Personal Data Download request

### PR DESCRIPTION
**Pull request by Mangesh-Khairnar**
Personal Data Deletion Request Documentation
Capture 003 - Personal Data Deletion - docs

Personal Data Download Request Documentation
FCapture 001 - Personal Data Download - docs

Index
Capture 002 - Setting Up - docs

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/setting-up/data/personal-data-deletion.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!--add breadcrumbs-->
+
+# Personal Data Deletion
+
+Personal data deletion tool enables a user to delete all the personal data a user has generated while using ERPNext.
+
+## Request to Delete Data Webform
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/data/personal-data-deletion.md]: rename this as 1. How to request deleting your data or something similar. remove the next heading. continue these as steps. Kindly refer to any manual page from selling, buying, stock etc to see how the pages are structured, this needs to be consistent with that

File: erpnext_com/www/docs/user/manual/en/setting-up/data/personal-data-deletion.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+## Email Verification
+
+This will sent a mail with a verification link of the data to the email address associated with the user.
+
+<img class="screenshot" alt="Verification Email" src="{{docs_base_url}}/assets/img/setup/personal-data-deletion-request/verification-email.png">
+
+Once the user clicks on the verification link.
+<img class="screenshot" alt="Confirmed Verification" src="{{docs_base_url}}/assets/img/setup/personal-data-deletion-request/confirmed-verification.png">
+
+## Personal Data Delete Request DocType
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/data/personal-data-deletion.md]: line 26 could be 2. How it works or something similar and the subsequent steps can be 2.1... etc
you don't need the escape character "\" when text is a heading

File: erpnext_com/www/docs/user/manual/en/setting-up/data/personal-data-download.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<!--add breadcrumbs-->
+
+# Personal Data Download
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/data/personal-data-download.md]: follow similar structure here also

File: erpnext_com/www/docs/user/manual/en/setting-up/data/index.txt [hunk 4]
```diff
@@ -1,3 +1,5 @@
 data-import
 bulk-rename
 download-backup
+personal-data-download
```
[the remark this answers is not in the document]
Comment (Mangesh-Khairnar) [reply erpnext_com/www/docs/user/manual/en/setting-up/data/index.txt]: moved to md

File: erpnext_com/www/docs/user/manual/en/setting-up/data/personal-data-deletion.md (hunk 1 above)
Comment (Mangesh-Khairnar) [reply erpnext_com/www/docs/user/manual/en/setting-up/data/personal-data-deletion.md]: added proper indexing

File: erpnext_com/www/docs/user/manual/en/setting-up/data-privacy/personal-data-deletion.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+<!--add breadcrumbs-->
+
+# Personal Data Deletion
+
+Personal data deletion tool enables a user to delete all the personal data a user has generated while using ERPNext.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/data-privacy/personal-data-deletion.md]: give examples, what kind of personal data?

File: erpnext_com/www/docs/user/manual/en/setting-up/data-privacy/personal-data-deletion.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    <img class="screenshot" alt="Deletion Request Success" src="{{docs_base_url}}/assets/img/setup/personal-data-deletion-request/deletion-request-success.png">
+
+3. This will sent a mail with a verification link of the data to the email address associated with the user.
+
+    <img class="screenshot" alt="Verification Email" src="{{docs_base_url}}/assets/img/setup/personal-data-deletion-request/verification-email.png">
+
+4. Once the user clicks on the verification link. A confirmation message will be displayed.
+    <img class="screenshot" alt="Confirmed Verification" src="{{docs_base_url}}/assets/img/setup/personal-data-deletion-request/confirmed-verification.png">
+
+### 2. How Deletion of users personal data works
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/data-privacy/personal-data-deletion.md]: How deleting user's personal data works

File: erpnext_com/www/docs/user/manual/en/setting-up/data-privacy/personal-data-deletion.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+<img class="screenshot" alt="Pending Verification" src="{{docs_base_url}}/assets/img/setup/personal-data-deletion-request/pending-verification.png">
+
+#### 2. Pending Approval
+This indicates that the user has verified the request via email link. This enables the option of "delete-data" for System Managers.
+
+<img class="screenshot" alt="Pending Approval" src="{{docs_base_url}}/assets/img/setup/personal-data-deletion-request/pending-approval.png">
+
+#### 3. Deleted
+This indicates that the System Manager has clicked on the "delete-data" button. The user personal data has been removed.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/data-privacy/personal-data-deletion.md]: does the button have a hyphen? Use proper casing

File: erpnext_com/www/docs/user/manual/en/setting-up/data-privacy/personal-data-download.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+<!--add breadcrumbs-->
+
+# Personal Data Download
+
+Personal data download tool enables user to automatically download all the personal data a user has generated while using erpnext.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/data-privacy/personal-data-download.md]: give couple of examples

**Pull request merged**